Applications/LLMEval/ContentView.swift

```diff
@@ -201,7 +201,6 @@ class LLMEvaluator {
     }
 
     func generate(prompt: String) async {
-        let startTime = Date()
```

Comment:
We can use the new generate loop & results to measure time (and be consistent with the command line tool)